go/cmd/vtbackup/cli/vtbackup.go

```diff
@@ -335,6 +340,14 @@ func takeBackup(ctx, backgroundCtx context.Context, topoServer *topo.Server, bac
 	if err != nil {
 		return fmt.Errorf("failed to initialize mysql config: %v", err)
 	}
+	ctx, cancelCtx := context.WithCancel(ctx)
+	backgroundCtx, cancelbackgroundCtx := context.WithCancel(backgroundCtx)
```

Comment:
As Matt mentions below, the context should always be cancelled.
```suggestion
	ctx, cancelCtx := context.WithCancel(ctx)
	defer cancelCtx()
	backgroundCtx, cancelbackgroundCtx := context.WithCancel(backgroundCtx)
	defer cancelbackgroundCtx()
```

Reply:
Fixed via 93d6c5273e0972f3ff56a942a30cadddca29fb6c